internal/host/host.go

```diff
@@ -354,10 +360,15 @@ func (m *Manager) IsRequireUserActionReset(h *models.Host) bool {
 	if swag.StringValue(h.Status) != models.HostStatusResetting {
 		return false
 	}
-	if !funk.Contains(manualRebootStages, h.Progress.CurrentStage) {
-		return false
+	if time.Since(time.Time(h.StatusUpdatedAt)) >= m.Config.ResetTimeout {
+		m.log.Infof("Host %s: exceeded reset timeout: %s", h.ID, m.Config.ResetTimeout.String())
```

Comment:
For both of these log messages please add more context (why is it interesting that the host exceeded timeout or passed reboot stage?  Because it's now moving to a pending-user-action.  Also please add the cluster ID.